WebDevStudios/wd_s
# Move .phpcs.xml to phpcs.xml.dist

### DESCRIPTION

This allows overriding it locally with a phpcs.xml file if needed. 
The path to the filed was removed from the composer commands as well, the default lookup order includes `phpcs.xml.dist`.

### OTHER


### STEPS TO VERIFY

Run `composer run lint` and `npm run format:php` and you should get the same output as expected before the change.
